src/Paket.Core/ProjectFile.fs

```diff
@@ -941,6 +941,9 @@ module ProjectFile =
                 project.ProjectNode.AppendChild analyzersNode |> ignore
             )
 
+    let touch (project:ProjectFile) =
+        if File.Exists(project.FileName) then
```

Comment:
It changes the logic previously present.
When the file wasn't present, it was created by the `save true project` call.
Please adjust accordingly.